broker/src/serve_tasks.rs

```diff
@@ -133,17 +133,12 @@ async fn get_results_for_task_stream(
     }
 
     // There must be a way to just use the stream of stream_results directly but I could not get it to work as the stream returned has a lifetime bound and this one somehow does not
```

Comment:
This comment is outdated with the change, no?

Reply:
Yes